pyaerocom/io/read_mscw_ctm.py

```diff
@@ -324,9 +332,11 @@ def preprocess_units(units, prefix=None):
         new_unit = units
         if units == '' and prefix == 'AOD': #
             new_unit = '1'
+        elif units == '' and prefix == 'AbsCoef':
+            new_unit = '1/m'
```

Comment:
```suggestion
            new_unit = 'm-1'
```
just for consistency, I think we should avoid /unit and rather use unit-1 nomenclature.